src/index.html

```diff
@@ -11,7 +11,209 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    name="forms"
+    >
```

Comment:
All attributes should have proper indentation if they start on a newline, fix it for all such tags, so it looks like this:
```suggestion
    <form
      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
      method="post"
      name="forms"
    >
```

Reply:
Again, fix your indentation using the [checklist](https://github.com/mate-academy/layout_html-form/blob/master/checklist.md),fix it for all the tags like this
```suggestion
    <form
      action="https://mate-academy-form-lesson.herokuapp.com/create-application"
      method="post"
      name="forms"
    >
```